src/main/java/net/smartcosmos/dao/objects/converter/ObjectResponseToObjectEntityConverter.java

```diff
@@ -0,0 +1,42 @@
+package net.smartcosmos.dao.objects.converter;
+
+import net.smartcosmos.dao.objects.domain.ObjectEntity;
+import net.smartcosmos.dto.objects.ObjectResponse;
+import net.smartcosmos.security.user.SmartCosmosUser;
+import net.smartcosmos.security.user.SmartCosmosUserHolder;
+
+import net.smartcosmos.util.UuidUtil;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.format.FormatterRegistrar;
+import org.springframework.format.FormatterRegistry;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author voor
```

Comment:
Basically agree. We even had the instruction to remove all `@author` annotations some time ago.

On the other hand that's ambivalent in my mind. Especially for open source projects I somehow appreciate the annotation for opportunities to meet the people behind a certain class in person (providing that it's a real name). It's right that the same information can be acquired from Git, but the implicit "at a glance" information in the Javadoc has its advantages as well.